### ISSUE TITLE
Feature to support nested git repos in workspace

### ISSUE THREAD

**Issue by L-Sypniewski** [koenigstag/git-changelists#2]
The extension doesn't work if I open a nested folder with VSCode (not the one with `.git` folder, but one below in the hierarchy).
In such case I get `Git repository not found. Try run 'git init' before using` error when trying to add a change list

**Comment by koenigstag** [comment]
Yes, such problem appears if you use an extension when a `.git` directory is in non-root but in nested folder.

This makes the extension currently unworkable for projects with this file structure.

As a single developer, I will try to see what I can do, in my free time.
It may take time to make a workaround for this case.
